doc/user_manual/rom.tex

```diff
@@ -85,6 +85,16 @@
             in order to use the option, the receiving \xmlNode{DataObject} should be of type
             \xmlNode{DataSet} with one of the indices being \texttt{\_ROM\_cluster}.
         \end{itemize}
+     \item \xmlNode{evaluationClusterChoice}, \xmlDesc{string, optional field}, one of \xmlString{first},
+        \xmlString{random}, or \xmlString{centroid}, determines, if \xmlAttr{grouping}$=cluster$, which 
+        strategy needs to be followed for the evaluation stage. If ``first'', the 
+        first ROM (representative segmented ROM),in each cluster, is considered to
+         be representative of the full space in the cluster (i.e. the evaluation is always performed
+         interrogating the first ROM in each cluster); If ``random'', a random ROM, in each cluster, 
+         is choosen when an evaluation is requested; If ``centroid'', a ROM ``trained" on the centroids 
+         information of each cluster is used for the evaluation (\nb ``centroid'' option is not 
+         available yet). 
```

Comment:
why adding the centroid option here? If it is not available...

Reply:
you are right...I remove it.